src/@types/react-diff-view/index.d.ts

```diff
@@ -1,7 +1,13 @@
 /* eslint react/no-multi-comp: 0, max-classes-per-file: 0 */
 
 declare module 'react-diff-view' {
-  type ChangeType = 'delete' | 'insert' | 'normal';
+  type ChangeType =
+    | 'delete'
+    | 'delete-eofnl'
```

Comment:
It might be good to add some comments about these since (I think?) they are not coming from `react-diff-view` but coming from our API instead.